postgres_composite_types/__init__.py

```diff
@@ -165,6 +147,9 @@ def database_forwards(self, app_label, schema_editor,
             "AS (%s)" % ', '.join(fields),
         )))
 
+        composite_type_created.send(self.Meta.model,
```

Comment:
Can we do that in the `Operation` itself? Create the type and then register it globally on all connections.